docs/how-to-use-gvfs.md

```diff
@@ -94,15 +94,15 @@ You can configure these properties in two ways:
 Please make sure to have the Gravitino Virtual File System runtime jar firstly, you can get it in
 two ways:
 
-1. [Maven repository](https://mvnrepository.com/): to download the runtime jar in the Maven central
-   repository, which names like `gravitino-filesystem-hadoop3-runtime-{version}.jar`.
+1. Download from the maven central repository: You can download the runtime jar which names like
+   `gravitino-filesystem-hadoop3-runtime-{version}.jar` from [Maven repository](https://mvnrepository.com/).
 2. Compile from the source code:
 
    Download the [Gravitino source code](https://github.com/datastrato/gravitino), and compile it
-   locally using the following command:
+   locally using the following command in the Gravitino source code directory:
 
     ```shell
-       ./${GRAVITINO_SOURCE_CODE_HOME}/gradlew :clients:filesystem-hadoop3-runtime:build -x test
+       ./gradlew :clients:filesystem-hadoop3-runtime:build -x test
```

Comment:
`compileDistribution` supports the filesystem jar?

Reply:
I think compileDistribution only support packaging the Gravitino server, this is a client-side library, so it is unrelated.

Reply:
I mean that  
```
./gradlew assembleTrinoConnector
```
I have dedicated command to compile trino connector. We should have similar command for file system in my opinion.

Reply:
It's not necessary, we already have a runtime uber jar which can be used.

Reply:
OK.